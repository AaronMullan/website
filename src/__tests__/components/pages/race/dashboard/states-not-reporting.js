import React from 'react'
import renderer from 'react-test-renderer'
import { useStaticQuery } from 'gatsby'
import StatesNotReporting from '~components/pages/race/dashboard/states-not-reporting'

describe('Components : Pages : Race : Dashboard : States not Reporting', () => {
  it('renders correctly', () => {
    const content = "<h2>Hi! I'm html content</h2><p>Me too :)</p>"
    const tree = renderer
      .create(<StatesNotReporting content={content} />)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
