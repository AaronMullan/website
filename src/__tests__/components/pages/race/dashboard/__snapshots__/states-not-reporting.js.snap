// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components : Pages : Race : Dashboard : States not Reporting renders correctly 1`] = `
<div
  className="container"
>
  <div
    className="iconContainer"
  >
    <img
      alt="Alert icon"
      src="test-file-stub"
    />
  </div>
  <div
    dangerouslySetInnerHTML={
      Object {
        "__html": "<h2>Hi! I'm html content</h2><p>Me too :)</p>",
      }
    }
  />
</div>
`;
